seed-job/template.yaml

```diff
@@ -35,7 +35,7 @@ objects:
                 containers: [
                   containerTemplate(
                     name: 'jnlp',
-                    image: 'registry.centos.org/dharmit/ccp-openshift-slave',
+                    image: '172.29.33.48:5000/nshaikh/ccp-jenkins-slave',
```

Comment:
Instead of using a dev's version of container image in every PR, can we have a Dockerfile in this repo and build `registry.centos.org/pipeline-images/ccp-openshift-slave` image using one copy? For dev environments we can always make sure that the dev environment is rebuilding the image and image from dev registry is used?

Reply:
+1 for having a respective Dockerfile in the repo.